tests/test_check_invariants.py

```diff
@@ -0,0 +1,41 @@
+"""
+Test suite for checking the functionality of check_invariants.
+"""
+
+import pytest
+
+from python_ta.contracts import PyTAContractError, check_invariants
+
+
+class Person:
+    """A custom data type that represents data for a person.
+
+    Representation Invariants:
+        - self.age >= 0
+    """
+
+    given_name: str
+    age: int
+
+    def __init__(self, given_name: str, age: int) -> None:
+        """Initialize a new Person object."""
+        self.given_name = given_name
+        self.age = age
+
+
+def test_no_errors() -> None:
+    """Checks that check_invariants does not raise an error when representation invariants are satisfied."""
+    person_obj = Person("Jim", 50)
+
+    try:
+        check_invariants(person_obj)
+    except Exception:
```

Comment:
Only catch errors that you're looking for: in this case, that would be `AssertionError`

Reply:
Hi Prof. Liu! Regarding the error type, I realize that `_check_invariants` only raises `PyTAContractErrors`, whereas `_instance_method_wrapper` (which is called when setting attributes) raises `AssertionErrors`. Is there a way I could change the behaviour of  `check_invariants` or `_check_invariants` to raise `AssertionErrors`?

Reply:
Good observation! Don't change the helper, but you should catch the PyTA error and then raise an AssertionError from the new function. Similar to what existing code does.

Think of the PyTA error as private, and AssertionError as the public error we should show users.

Reply:
Now that you've made this change, you still need to change `Exception` to `AssertionError` here